<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.my.automation.ui.SFLoginTest2" tests="2" time="0.005" ignored="1" failures="0" timestamp="13 Sep 2018 11:45:41 GMT" skipped="1" hostname="D1ML14617" errors="1">
  <testcase name="beforeClass" time="0.000" classname="com.my.automation.ui.SFLoginTest2">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.my.automation.basepageobject.BasePageObject.<init>(BasePageObject.java:20)
	at com.my.automation.pageobjects.UIBasePageObject.<init>(UIBasePageObject.java:10)
	at com.my.automation.pageobjects.SFLoginPage.<init>(SFLoginPage.java:21)
	at com.my.automation.ui.UIBaseTest.loadPageObjects(UIBaseTest.java:30)
	at com.my.automation.ui.UIBaseTest.beforeClass(UIBaseTest.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeClass -->
  <testcase name="loginToSF" time="0.005" classname="com.my.automation.ui.SFLoginTest2">
    <skipped/>
  </testcase> <!-- loginToSF -->
  <testcase name="beforeTest" classname="com.my.automation.base.test.BaseTest">
    <skipped/>
  </testcase> <!-- beforeTest -->
</testsuite> <!-- com.my.automation.ui.SFLoginTest2 -->
